src/ServiceManagement/Storage/StorageManagement/Microsoft.WindowsAzure.Management.Storage.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.WindowsAzure.Management.Storage
     -->
     <SdkNuGetPackage Include="Microsoft.WindowsAzure.Management.Storage">
-      <PackageVersion>5.1.1</PackageVersion>
+      <PackageVersion>5.1.2</PackageVersion>
```

Comment:
Since this update bump up the version header from 2014-10-01 to 2016-03-01, I think the major version of the llibrary should be updated, such as 6.0.0.  (Or at least, it should be 5.2.0)